Index and show for book test

NicolasLeutwyler opened the pull request:
# Summary

* The Index and Show for book tested.

# Trello Card

* [Card.](https://trello.com/c/PMyOZFzx/25-crear-tests-para-rutas-index-y-show-de-book)

mdesanti commented on .travis.yml:
```diff
@@ -25,6 +25,7 @@ before_script:
 
 script:
   - bundle exec rspec spec -fd
+  - bundle exec rspec spec/controllers/api/v1/books_controller.rb
```
There is no need to add this. By executing `bundle exec rspec spec -fd` you are running all tests under the `spec` folder

NicolasLeutwyler commented:
I think it's done, I've modified the **spec/controllers/api/v1/books_controller.rb** file name to **spec/controllers/api/v1/books_controller_spec.rb** and removed that useless travis line.
